pkg/preparer/orchestrate.go

```diff
@@ -68,7 +68,7 @@ func WatchForPodManifestsForNode(nodeName string, consulAddress string, hooksDir
 
 // no return value, no output channels. This should do everything it needs to do
 // without outside intervention (other than being signalled to quit)
-func handlePods(hooks *pods.HookDir, podChan <-chan pods.PodManifest, quit <-chan struct{}, logger logging.Logger) {
+func handlePods(hooks *pods.HookDir, podChan <-chan pods.PodManifest, quit <-chan struct{}, logger logging.Logger, store *intent.Store) {
```

Comment:
I was actually considering that refactor as part of this commit. It'll also be useful for the reality store. @petertseng thoughts? I don't want to block your work on this refactor so we could delay it until then.